code/src/App.js

```diff
@@ -1,9 +1,85 @@
-import React from 'react'
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable react/react-in-jsx-scope */
+import './reset.css';
+import './styles.css';
+import './dark-theme.css';
+
+import { useState, useEffect } from 'react';
+
+import Message from './components/Message';
+import Submission from './components/Submission';
+
+export default function App() {
+  const [messages, setMessages] = useState([]);
+  const [areLiked, setAreLiked] = useState([]);
+  const [needsUpdating, setNeedsUpdating] = useState(false);
+
+  function themeSwitcher() {
+    if (
+      window.matchMedia
+      && window.matchMedia('(prefers-color-scheme: light)').matches
+    ) {
+      document.body.classList.toggle('is-forced-dark');
+    } else if (
+      window.matchMedia
+      && window.matchMedia('(prefers-color-scheme: dark)').matches
+    ) {
+      document.body.classList.toggle('is-forced-light');
+    }
+  }
+
+  useEffect(() => {
+    // Ropa på api asynkront
+    const url = 'https://happy-thoughts-technigo.herokuapp.com/thoughts';
+    fetch(url, {
+      method: 'GET'
+    })
+      .then((raw) => {
+        return raw.json();
+      })
+
+      .then((formatted) => {
+        setMessages(formatted);
+        if (localStorage.getItem('are-liked')) {
+          setAreLiked(JSON.parse(localStorage.getItem('are-liked')));
+        } else {
+          console.log('localStorage finns inte');
+        }
+      })
+
+      .catch((e) => {
+        console.log('Something went wrong', e);
+        setNeedsUpdating(true);
```

Comment:
I'm not sure you need this state property - if something will go wrong with request, it will never be resolved, therefore old set of data will still be valid and presented to the user